boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu1.dts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2024 NXP
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <nxp/nxp_mcxn94x.dtsi>
+#include "frdm_mcxn947.dtsi"
+
+/ {
+	model = "NXP FRDM_N94 board";
+	compatible = "nxp,mcxn947", "nxp,mcx";
+
+	cpus {
+		/delete-node/ cpu@0;
+	};
+
+	chosen {
+		zephyr,sram = &sramg;
+		zephyr,flash = &flash;
+		zephyr,flash-controller = &fmu;
+		zephyr,code-partition = &cpu1_partition;
+		zephyr,console = &flexcomm2_lpuart2;
+		zephyr,shell-uart = &flexcomm2_lpuart2;
+	};
+};
+
+&flexcomm2 {
+	status = "okay";
+};
+
+&flexcomm2_lpuart2 {
+	status = "okay";
+};
+
+&edma0 {
+	status = "okay";
+};
+
+&mbox {
+	status = "okay";
+};
+
+&w25q64jvssiq {
```

Comment:
aren't some of these devices being status=okay going to conflict with what devices are status=okay for cpu0 ? I don't think most of these drivers have been written with the intention of two different cores doing i/o doing their own i/o and management of the peripherals

Reply:
I think I was adding them because some tests here in CI failed.
Because they try to run some flash example on standalone CPU1.
I removed them, let see what CI will tell.

Reply:
Yes the flash is required, otherwise the test `frdm_mcxn947/mcxn947/cpu1 drivers.flash.common.default` fails.

Reply:
that's because the Devicetree has a "fixed-partitions" node with a storage partition on the external winbond flash node, is this something we need to support having enabled on both cores simultaneously?  my opinion is to move this definition out of the common board .dtsi and into the cpu0 specific file

Reply:
Thanks for the tip here, I moved the storage partition to cpu0 only and CI passes now!